packages/framework-core/src/services/graphql/websocket-protocol/graphql-websocket-protocol.ts

```diff
@@ -104,8 +121,25 @@ export class GraphQLWebsocketHandler {
 
     this.logger.debug('Operation finished. Sending DATA:', result)
     // It was a query or mutation. We send data and complete the operation
-    await this.sendToConnection(connectionID, new GraphQLData(message.id, result))
-    await this.sendToConnection(connectionID, new GraphQLComplete(message.id))
+    await this.connectionManager.sendMessage(this.config, connectionID, new GraphQLData(message.id, result))
+    await this.connectionManager.sendMessage(this.config, connectionID, new GraphQLComplete(message.id))
+  }
+
+  private async augmentEnvelope(
+    connectionID: string,
+    envelope: GraphQLRequestEnvelope,
+    message: GraphQLStart
+  ): Promise<GraphQLRequestEnvelope> {
+    const connectionData = await this.connectionManager.fetchData(this.config, connectionID)
```

Comment:
Here we fetch the connection data associated to this socket connection. Then we set the user information, that is among the connection data, to the envelope